mllib/src/main/scala/org/apache/spark/mllib/stat/MultivariateOnlineSummarizer.scala

```diff
@@ -52,7 +52,7 @@ class MultivariateOnlineSummarizer extends MultivariateStatisticalSummary with S
   private var totalCnt: Long = 0
   private var totalWeightSum: Double = 0.0
   private var weightSquareSum: Double = 0.0
-  private var weightSum: Array[Double] = _
+  private var currWeightSum: Array[Double] = _
```

Comment:
Nit: I don't think the rename was necessary, but it is OK

Reply:
done!

Reply:
Nevermind, it looks like the build failed because the private variable conflicts with the public function that was defined:

  /**
   * Sum of weights.
   */
  override def weightSum: Double = totalWeightSum

I think this may be the best name for the public variable so I would prefer to keep it.  The private variable now follows the naming convention of the other private array variables so I think this makes sense.